<?xml version="1.0"?>
<launch>

  <!-- sample file, copy this to /etc/ros/fuerte/eddiebot.launch -->

  <param name="covi_node/gyro_scale_correction" value="1.0"/>
  <param name="covi_node/odom_angular_scale_correction" value="1.0"/>

  <include file="$(find covi)/launch/minimal.launch">
    <arg name="urdf_file" value="$(find xacro)/xacro '$(find covi_description)/urdf/covi_realsense.urdf.xacro'"/>
  </include>

</launch>
